axelar-chains-config/info/testnet.json

```diff
@@ -1872,6 +1893,11 @@
       "gasOptions": {
         "gasLimit": 8000000
       },
+      "onchainGasEstimate": {
```

Comment:
also check for fraxtal

Reply:
Yeah it's still using bedrock, but we haven't deployed it yet.